mobile/src/main/java/com/wedriveu/mobile/app/MainActivity.java

```diff
@@ -77,4 +79,37 @@ public void dismissProgressDialog() {
         View progressDialog = findViewById(R.id.progress_dialog);
         progressDialog.setVisibility(View.GONE);
     }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        mLocationService.checkSettings();
```

Comment:
I just marked it as low priority for the project. Also, I think the user experience is not compromised.